VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegmentImpl.java

```diff
@@ -79,20 +76,15 @@ private void setCustomProperties() {
      * @param email  -  email of the user
      * @param username  -  Username/email that the user uses for signing in
      *
-     * @return
+     * @return Returns a {@link Traits} object populated with provided user info
```

Comment:
The "Returns" at the start is redundant as the JavaDocs add that automatically. The sentence should still start capitalized though.